File: Day36/zero_one_triangle.py

"""Function to print a 0-1 equilateral triangle pattern"""

def print_triangle(rows):

    """Loop through each row f the  triangle"""
    for i in range(1,rows+1):
        #print spaces
        print(" " * (rows-i),end="")
        #Print 0-1 pattern
        for k in range(1,i+1):
            print((i+k)%2,end=" ")
        print()

input_rows = int(input("Enter the number of row: "))

print_triangle(input_rows)

File: Day36/zeroOneTriangle.js
//Function to print a 0-1 equilateral triangle pattern
function printTriangle(rows){
    for(let i= 0; i< rows; i++){
        //print spaces
        let row ='';
        for(let j= 0; j<= rows-i-1; j++){
            row += ' '; 
        }


        //Print 0-1 pattern
        for (let k = 0; k <= i; k++){
            row += (i+k)%2 + ' ';
        }
        console.log(row); 
    }
}

input_rows = 5;
printTriangle(input_rows);
